spec/dummy/docker-compose.yml

```diff
@@ -1,6 +1,6 @@
 services:
   strata-dummy-database:
-    image: postgres:14-alpine
+    image: postgres:16-alpine
```

Comment:
I accidentally merged this in here, was supposed to be part of a different PR https://github.com/navapbc/flex-sdk/pull/227